tests/integration-tests/src/test/resources/multicast_topic.xml

```diff
@@ -85,13 +85,13 @@ under the License.
 
       <security-settings>
          <security-setting match="#">
-            <permission type="createNonDurableQueue" roles="a,b"/>
-            <permission type="deleteNonDurableQueue" roles="a,b"/>
```

Comment:
This was important for the tests, as it ensures that the security locks down for the other address's and doesn't rely on the catch all credentials, can you add back